src/chapters/scope/introduction.rst

```diff
@@ -0,0 +1,144 @@
+Introduction
+============
+
+.. index:: ! scope
+
+Where variables are declared and initialized in the code can effect whether or not they can be used.
+**Scope** refers to the ability to access and use the variable.
```

Comment:
Should it read: "**Scope** refers to the ability to access and use a variable." or "**Scope** refers to the ability to access and use the variable."?

Reply:
I like the first one!